packages/api/src/repository/ResultRequest.ts

```diff
@@ -55,9 +55,16 @@ export class ResultRequestRepository {
   }
 
   async insert (resultRequest: Omit<ResultRequestDbObject, '_id'>) {
-    const response = await this.collection.insertOne(resultRequest)
+    if (this.isValidResultRequest(resultRequest)) {
+      const response = await this.collection.insertOne(resultRequest)
 
-    return this.normalizeId(response.ops[0])
+      return this.normalizeId(response.ops[0])
+    } else {
+      console.error(
+        'Error inserting result request: Validation Error',
+        resultRequest
+      )
+    }
```

Comment:
This clause allows the function to never return explicitly, which is an ugly thing to do for asynchronous functions hehehe